test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -54,6 +105,27 @@ void test_ucp_mmap::test_rkey_management(entity *e, ucp_mem_h memh, bool is_dumm
     }
     ASSERT_UCS_OK(status);
 
+    bool have_rma    = resolve_rma(e, rkey);
```

Comment:
is it ok to define variables in mid of function?

Reply:
in C++, yes